thrust/cmake/ThrustBuildCompilerTargets.cmake

```
@@ -131,6 +131,12 @@ function(thrust_build_compiler_targets)
     )
   endforeach()
 
+  if (THRUST_DISPATCH_TYPE STREQUAL "Force32bit")
+    list(APPEND cxx_compile_definitions "THRUST_FORCE_32BIT_OFFSET_TYPE")
```

I would rename to add another underscore.

```suggestion
    list(APPEND cxx_compile_definitions "THRUST_FORCE_32_BIT_OFFSET_TYPE")
```